osu.Game.Rulesets.Mania/Difficulty/ManiaPerformanceCalculator.cs

```diff
@@ -55,30 +58,212 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
             return new ManiaPerformanceAttributes
             {
                 Difficulty = difficultyValue,
-                Total = totalValue
+                Total = totalValue,
+                EstimatedUR = estimatedUR
             };
         }
 
         private double computeDifficultyValue(ManiaDifficultyAttributes attributes)
         {
-            double difficultyValue = Math.Pow(Math.Max(attributes.StarRating - 0.15, 0.05), 2.2) // Star rating to pp curve
-                                     * Math.Max(0, 5 * scoreAccuracy - 4) // From 80% accuracy, 1/20th of total pp is awarded per additional 1% accuracy
-                                     * (1 + 0.1 * Math.Min(1, totalHits / 1500)); // Length bonus, capped at 1500 notes
+            double difficultyValue = Math.Pow(Math.Max(attributes.StarRating - 0.15, 0.05), 2.2); // Star rating to pp curve
+
+            difficultyValue *= Math.Max(1.2 * Math.Pow(SpecialFunctions.Erf(300 / estimatedUR), 1.6) - 0.2, 0);
```

Comment:
Please help clarify what is this change with graphs

Reply:
added a desmos link